.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
`.idea` says to exclude a file named `.idea`. If you want to exclude a directory, add `/` at the end (like `.idea/`).